public/index.js

```diff
@@ -70,6 +80,7 @@ Vue.createApp({
 				.finally(()=>{
 					this.showInstructions = false;
 					this.isLoading = false;
+					this.timer = setTimeout(this.resetState, resetTimeoutLength);
```

Comment:
This can be removed for simplicity - brings all the timer-manipulation to the same location.
"reset timer when something is typed or when the result is loaded" is almost equal to "reset timer when something is typed", specially when the timer is 3 minutes.